packages/e2e-tests/fixtures/blocks/core__gallery__deprecated.html

```diff
@@ -0,0 +1,10 @@
+<!-- wp:core/gallery {"ids":[null,null],"columns":2} -->
```

Comment:
There's multiple deprecation, but I only see one being added here?

Reply:
Yes, it's time-consuming to backport all those missing fixtures.